completions/ssh

```diff
@@ -431,11 +431,29 @@ _sftp()
 # shellcheck disable=SC2089
 _comp_cmd_scp__path_esc='[][(){}<>"'"'"',:;^&!$=?`\\|[:space:]]'
 
-# Complete remote files with ssh.  If the first arg is -d, complete on dirs
-# only.  Returns paths escaped with three backslashes.
+# Complete remote files with ssh. Returns paths escaped with three backslashes
+# (unless -l option is provided).
+# Options:
+#     -d  Complete on dirs only.
+#     -l  Return paths escaped with one backslash instead of three.
 # shellcheck disable=SC2120
 _comp_xfunc_ssh_scp_remote_files()
 {
+    local dirs_only
+    local less_escaping
```

Comment:
When `shopt -s localvar_inherit` is set, the declaration without values will inherit the values in the outer scope. We need to initialize them with any values. One way is to initialize them with empty values and later reference them as `[[ $dirs_only ]]`.

```suggestion
    local dirs_only=
    local less_escaping=
```

Another way is to initialize them with `false` and later reference them as `"$dirs_only"`.

```suggestion
    local dirs_only=false
    local less_escaping=false
```

We are recently using the `true`/`false` pair instead of the non-empty/empty pair.